air/src/lib.rs

```diff
@@ -273,9 +273,9 @@ impl PublicInputs {
 
 impl vm_core::ToElements<Felt> for PublicInputs {
     fn to_elements(&self) -> Vec<Felt> {
-        let mut result = self.program_info.to_elements();
-        result.append(&mut self.stack_inputs.to_vec());
+        let mut result = self.stack_inputs.to_vec();
         result.append(&mut self.stack_outputs.to_vec());
+        result.append(&mut self.program_info.to_elements());
         result
```

Comment:
Does `miden-base` or `miden-node` currently assume the previous structure of `PublicInputs`'s elements representation?

cc @Mirko-von-Leipzig @PhilippGackstatter

Reply:
Good point @plafer ! I made the change as it seemed cleanest on the VM side but if it complicates things elsewhere then we can revert and update things.

Reply:
We don't use this type in miden-base directly. This seems to be only exported from `miden-air` but not `miden-verifier` or `miden-prover`, which we depend on, so in that sense we do not assume the element structure.